sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -79,6 +79,11 @@ static const struct snd_soc_acpi_link_adr tgl_rvp[] = {
 	{}
```

Comment:
For the commit message.  The indentation in commit message seems a bit of, maybe just use: 
```
ASoC: Intel: common: add mach entry for tgl_max98373_rt5682

Update tgl mach table with: Maxim98373 Amp and ALC5682 hp codec. Both 
of the codecs are on I2S bus.
```

Reply:
Thank you @kv2019i for the suggestion. I have updated now.